packages/babel-plugin/src/utils/evaluate-expression.tsx

```diff
@@ -133,6 +134,24 @@ const evaluateIdentifierBindingMemberExpression = (
   return createResultPair(value, updatedMeta);
 };
 
+const evaluateNamespaceImportExpression = (accessPath: t.Identifier[], meta: Metadata) => {
+  const exportName = accessPath[0].name;
+  const updatedMeta: Metadata = { ...meta };
+  const { file } = meta.state;
+  const result =
+    exportName === 'default' ? getDefaultExport(file) : getNamedExport(file, exportName);
+
+  if (result) {
+    updatedMeta.parentPath = result.path.parentPath;
```

Comment:
I think I'm suppose to set `ownPath` instead of `parentPath`. I tried it and it didn't make a difference we hit identities from other paths. I might be doing something wrong here. Will look into it more.

Reply:
Yeah, I'd expect both to be set since they're used to resolve bindings

Reply:
Yeah, I'd expect both to be set since they're used to resolve bindings